openghg/types/_enum.py

```diff
@@ -73,3 +73,7 @@ class DataTypes(Enum):
     NPL = "NPL"
     OPENGHG = "OPENGHG"
     INTEM = "INTEM"
+
+
+class BoundaryConditions(Enum):
+    BOUNDARY_CONDITIONS = "Boundary_Conditions"
```

Comment:
As per other suggestions for source_format:
```suggestion
    OPENGHG = "OPENGHG"
```